### ISSUE TITLE
Help to identify the ploidy

### ISSUE THREAD

**Issue by Huyuxi08** [schatzlab/genomescope#92]
Hi, 
Thanks for developing this great tool.

I'm trying to use GenomeScope 2 to identify the ploidy of two plant samples.
The result of sample A indicates that it's a tetraploid. But the result of sample B is not that typical. If sample B is a tetraploid, is the reads duplication rate of these two samples that cause the difference in their GenomeScope plots? (The sequencing data of the two samples is the same, about 20G. And the genome size is 950 Mb)
![5F6FA29D-34B3-4C53-A5EE-ED42772C88B4](https://user-images.githubusercontent.com/60483748/221532396-eb22b465-4890-4e96-8dd7-75e10da7619c.jpeg)

**Comment by mschatz** [comment]
I agree sample A looks tetraploid. B is unusual. Is it from a single plant
or a mixture? If it is mixed that can really complicate the analysis. Have
you tried smudgepairs? That can sometimes help as well

Good luck!

Mike
